main_server/local_server/server_admin/server_admin_javascript.js

```diff
@@ -1569,6 +1569,8 @@ function BMLT_Server_Admin()
                                 this.createOneMeetingNode(single_meeting_div, this.m_search_results[c]);
                                 meeting_changed = true;
                             };
+
+                            break;
```

Comment:
There's no reason to keep iterating after the meeting is found, so just `break`.